examples/servo/main.c

```diff
@@ -0,0 +1,25 @@
+#include "../../libtock/interface/syscalls/servo_syscalls.h"
+#include <libtock-sync/services/alarm.h>
+#include <libtock/tock.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(void) {
+  // Checks if the driver exists and, if not, returns -1.
+  if (!libtock_servo_exists()) {
+    printf("There is no available servo\n");
+    return -1;
+  }
+
+  uint16_t angle;
```

Comment:
```suggestion
  uint16_t angle = 0;
```

It is good practice to initialise variables.